bootstrap/15-locale.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Copyright (C) 2017 Denis Mosolov <denismosolov@cusdeb.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# Set timezone
+# https://bugs.launchpad.net/ubuntu/+source/tzdata/+bug/1554806
```

Comment:
There are a comment and a related link, but still I think there is something missing. Probably we should provide a summary of the bug and after this provide the link for the ones who are interested in the details. What do you think?

Reply:
I think that's fine. I don't want to overtext it.